src/bp/diag/index.ts

```diff
@@ -48,6 +48,7 @@ export const ENV_VARS = [
   'BPFS_STORAGE',
   'NATIVE_EXTENSIONS_DIR',
   'REDIS_URL',
+  'REDIS_CHANNEL_PREFIX',
```

Comment:
We should prefix all future environment variables with "BP_" , so it's clear it is being used by us. The diag tool already extracts all variables starting with "BP_" and it should not be necessary to add it to "ENV_VARS" here (this list is to support older variables to prevent breaking changes)